network/api.go

```diff
@@ -149,9 +155,17 @@ func NewHandler(driver Driver) *Handler {
 
 func (h *Handler) initMux() {
 	h.HandleFunc(capabilitiesPath, func(w http.ResponseWriter, r *http.Request) {
-		fmt.Fprintln(w, defaultScope)
+		res, err := h.driver.GetCapabilities()
+		if err != nil {
+			msg := err.Error()
+			sdk.EncodeResponse(w, NewErrorResponse(msg), msg)
+		}
+		if res != nil {
+			sdk.EncodeResponse(w, res, "")
+		} else {
```

Comment:
By changing the `interface` a plugin *must* implement a `GetCapabilities()` method and return a valid `*CapabilitiesResponse`.

Therefore, I'd change this to throw and error `if res == nil` and `EncodeResponse` otherwise.

To make this easier on users, let's add `LocalScope` and `GlobalScope` as constants, so a plugin only needs to do `return &CapabilitiesResponse{Scope:network.LocalScope}`